processor/dnslookupprocessor/config.go

```diff
@@ -42,10 +44,8 @@ type Config struct {
 
 // LookupConfig defines the configuration for forward/reverse DNS resolution.
 type LookupConfig struct {
```

Comment:
```suggestion
type lookupConfig struct {
```

I think it doesn't need to be exported as well.